pkg/config/mq_sink_protocol.go

```diff
@@ -31,13 +30,14 @@ const (
 	ProtocolMaxwell
 	ProtocolCanalJSON
 	ProtocolCraft
+	ProtocolOpen
 )
 
 // FromString converts the protocol from string to Protocol enum type.
-func (p *Protocol) FromString(protocol string) {
+func (p *Protocol) FromString(protocol string) error {
 	switch strings.ToLower(protocol) {
 	case "default":
-		*p = ProtocolDefault
+		*p = ProtocolOpen
```

Comment:
now the input and output by `String()` is different, is this expected? For example, backup the output and apply it to a cluster of lower version

Reply:
i will fix it in next pr. I will refine the meta dta from  low version cluster.